docs/source/templates/taxonomy.md

```diff
@@ -26,40 +44,184 @@ Perform classification tasks within the context of a defined taxonomy or hierarc
+
+For example: 
+
+```html
+<View>
+  <Text name="text" value="$text"/>
+  <Taxonomy name="taxonomy" toName="text" apiUrl="https://cities-nu.vercel.app/full" />
+</View>
+```
+
+The remote taxonomy must use JSON with items in the following format: 
+
+| Property | Description |
+| --- | --- |
+| `items` | Required. The JSON resource should be structured as an object with the key `items`. See below for examples. |
+| `value` | Required. This is what the user sees as an option to select. If you do not include an `alias` property, the value is exported in the annotation results. |
+| [`alias`] | If included, the `alias` replaces the `value` property in the annotation results. The `alias` is not displayed in the labeling interface. This is useful when you have internal identifiers for your data.  |
+| [`children`] | Nested values within the taxonomy hierarchy. Use this when defining your taxonomy in a single JSON-formatted file. |
+| [`isLeaf`] | Boolean value. Use this instead of `children` when working with an API taxonomy. The default is `true`. When explicitly set to false, the node is treated as a parent node. [See below for more information](#API-taxonomies).  |
+| [`hint`] | This string will appear as a tooltip to the user when they hover their cursor over the value. |
+
+
+### Flat file format
+
+The basic requirements are to use JSON formatting, wrap the taxonomy structure in an `items` object, and include `value` properties for every item. 
+
+In this example, you are using `children` to specify child nodes. All values are loaded in a single request. 
+
+
+```json
+{
+    "items": [
+        {
+            "alias": "archaea01",
+            "hint": "Single-celled organisms",
+            "value": "Archaea"
+        },
+        {
+            "alias": "bacteria01",
+            "hint": "Prokaryotic microorganisms",
+            "value": "Bacteria"
+        },
+        {
+            "alias": "eukarya01",
+            "hint": "Basically everything else",
+            "value": "Eukarya",
+            "children": [
+                {
+                    "alias": "eukarya01_b1",
+                    "value": "Human"
+                },
+                {
+                    "alias": "eukarya01_b2",
+                    "value": "Opossum"
+                },
+                {
+                    "alias": "eukarya01_b3",
+                    "value": "Extraterrestrial"
+                }
+            ]
+        }
+    ]
+}
 ```
-Nest choices under a specific [Choice](/tags/choice.html) tag to create layers in the taxonomy.
+
+
+### API taxonomies 
+
+When using this format, child nodes are only loaded when requested. Parent nodes are specified using `"isLeaf"=false` and child nodes are called through the `path` parameter.
+
+For example, `taxonomy_file?path=node1` where `node1` is the alias (if specified) or the value (if no alias is specified). Therefore you must ensure that your API supports the `path` parameter. 
```

Comment:
To avoid any confusion
```suggestion
For example, `taxonomy_api_url?path=node1` where `node1` is the alias (if specified) or the value (if no alias is specified). Therefore you must ensure that your API supports the `path` parameter. 
```